conjure-java-core/src/integrationInput/java/com/palantir/product/EteBinaryServiceEndpoint.java

```diff
@@ -74,7 +75,7 @@ public void handleRequest(HttpServerExchange exchange) throws IOException {
                 if (result.isPresent()) {
                     BinarySerializers.serialize(result.get(), exchange);
                 } else {
-                    exchange.setStatusCode(204);
+                    exchange.setStatusCode(StatusCodes.NO_CONTENT);
```

Comment:
This is easier to read, nobody will have to look up 204. Compiled bytecode is identical.